edith-step/src/App.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import "./App.css";
+import ReceiptInput from './ReceiptInput';
+//import "./App.css";
```

Comment:
Do we not need this css file ?

Reply:
It's just the one that displays the spinning logo so we'll need it at some point but it's not tailored to it rn